latest_cov_build.py

```diff
@@ -0,0 +1,26 @@
+import tarfile
```

Comment:
Make sure there are no errors when running `flake8` on this file (and fix any error that will come up)

Reply:
Also, please divide the file in three sections: 1) Retrieve task from Index; 2) Retrieve artifact; 3) Unpack artifact

To do this either define functions and call them or do something like this:
```
# Retrieving task from index
CODE HERE

# Retrieving artifact from Queue
CODE HERE

# Extracting artifact
CODE HERE
```